lib/data_update_scripts/20200826141447_remove_orphaned_page_views_by_user.rb

```diff
@@ -0,0 +1,14 @@
+module DataUpdateScripts
+  class RemoveOrphanedPageViewsByUser
+    def run
+      # Delete all Collections belonging to Users that don't exist anymore
+      ActiveRecord::Base.connection.execute(
+        <<~SQL,
+          DELETE FROM page_views
```

Comment:
Makes sense, page views belong to the article, not to the user. The reason why I went with delete is that at the present moment they are deleted: 

https://github.com/forem/forem/blob/1336d8c439d0d48867ed0b76fc9ef0013bf0c9e0/app/models/user.rb#L159

at they have been at least since 7 months ago when we merged https://github.com/forem/forem/pull/5624

I'm fine with switching the relation between the two tables but the catch is: aren't we going to have counts that are off for all the articles that have been posted since?

I'm okay either way

Reply:
We even explicitly delete them when we use the `Users::Destroy` class 

https://github.com/forem/forem/blob/1336d8c439d0d48867ed0b76fc9ef0013bf0c9e0/app/services/users/delete_activity.rb#L21

Reply:
> aren't we going to have counts that are off for all the articles that have been posted since?

I wouldn't say they were off, they would just have a little boost from the change in behavior. Part of me thinks just keep the status quo bc its easy and the way we have been doing it and I dont see "tons" of users getting deleted that would kill article page views. 🤔  I'm torn. I leave it up to you if you want more input on this. I can live with it either way

Reply:
I'm going to cc @joshpuetz, @vaidehijoshi and @benhalpern here to see if they have a preference about this and `rating_votes` [below](https://github.com/forem/forem/pull/10011#discussion_r477458751)

Reply:
I'm with @mstruve  on this one, the page view doesn't go away because the user does. And if we can't tie it back to an actual user, it's not PII so also not problematic from a compliance/data perspective. If people agree to change it I think the sooner the better.

Reply:
i concur that we should keep the page views around, as they represent an entirely different "entity" (the page was viewed by _someone_, and we don't really care _who_ for the purpose of counting it). we may want to leave some comments about _why_ we explicitly don't want to delete the page view AND the rating so that we don't accidentally revert this.